### PR TITLE
CustomerIO config for testing env

### PR DESCRIPTION
**Pull request by sinaahmadpour**

### CODE REVIEW

File: src/Traits/BranchDeployForgeInputs.php [hunk 1]
```diff
@@ -37,4 +37,16 @@ protected function getPhpVersion(): string {
...
         return str_replace('.', '', $this->getPhpVersion());
     }
+
+    protected function getCustomerIoConfig(string $key): string {
+        $configs = [
+            'site_id' => getenv('CUSTOMERIO_SITE_ID'),
+            'api_key' => getenv('CUSTOMERIO_API_KEY'),
+            'app_api_key' => getenv('CUSTOMERIO_APP_API_KEY'),
+            'webhook_url' => getenv('CUSTOMERIO_WEBHOOK_URL'),
+            'webhook_internal_key' => getenv('CUSTOMERIO_WEBHOOK_INTERNAL_KEY'),
```
Comment (BrianJunk) [inline src/Traits/BranchDeployForgeInputs.php:47]: I've updated the access, can you view this page https://github.com/timberhubcom/envs/settings/secrets/actions

Comment (sinaahmadpour) [reply src/Traits/BranchDeployForgeInputs.php:47]: Screenshot 2024-03-25 at 09 17 48
It seems I still don't have access.

Comment (BrianJunk) [reply src/Traits/BranchDeployForgeInputs.php:47]: try again?

Comment (sinaahmadpour) [reply src/Traits/BranchDeployForgeInputs.php:47]: It works now, Thank you 🙏🏻

**Review by b-amar — APPROVED** [review]
LGTM

File: src/Traits/BranchDeployForgeInputs.php (hunk 1 above)
Comment (sinaahmadpour) [reply src/Traits/BranchDeployForgeInputs.php:47]: Action secrets have been added

**Review by BrianJunk — APPROVED** [review]

**Pull request merged**